documentation/jetty/modules/programming-guide/pages/server/http.adoc

```diff
@@ -814,6 +814,29 @@ include::code:example$src/main/java/org/eclipse/jetty/docs/programming/server/ht
 If the resource is not found, `ResourceHandler` will not return `true` from the `handle(\...)` method, so what happens next depends on the `Handler` tree structure.
 See also <<handler-use-default,how to use>> `DefaultHandler`.
 
+[[handler-use-conditional]]
+==== ConditionalHandler
+
+`ConditionalHandler` is an abstract `Handler` that matches conditions about the request, and allows subclasses to handle the request differently depending on whether the conditions have been met or not.
+
+Subclasses must override methods:
+
+* `boolean onConditionsMet(Request, Response, Callback)`, to handle the request when the conditions are met.
+* `boolean onConditionsNotMet(Request, Response, Callback)`, to handle the request when the conditions are not met.
```

Comment:
You should mention the sub types:
 + ElseNext
 + SkipNext
 + DontHandle
 + Reject

which implement one or both of these methods with common behaviours

Reply:
Done.